packages/components/src/components/Dropzone/Dropzone.test.tsx

```diff
@@ -74,6 +74,7 @@ describe("<Dropzone />", () => {
   });
 
   it("calls onCancel when Cancel button is clicked", async () => {
+    const user = userEvent.setup();
```

Comment:
Nit: we could move this `const user = userEvent.setup();` to the start of the test so we wouldn't have to repeat it in multiple test cases.